concept/type/impl/AttributeTypeImpl.java

```diff
@@ -79,6 +81,11 @@ public final boolean isKeyable() {
         return new AttributeTypeImpl.Remote(transaction, getLabel(), isRoot());
     }
 
+    @Override
+    public boolean isAttributeType() {
```

Comment:
Let's push this up into `AttributeType`.

There are other examples - apply the change to all of them!

Reply:
Fixed: https://github.com/graknlabs/client-java/pull/211/commits/128016426240102c1db53eff8c33755138a8b0ad